src/main/java/com/datatorrent/netlet/OptimizedEventLoop.java

```diff
@@ -75,9 +75,9 @@ public boolean add(SelectionKey key)
       }
       keys[pos++] = key;
       if (pos == keys.length) {
-        SelectionKey[] keys = new SelectionKey[this.keys.length << 1];
-        System.arraycopy(this.keys, 0, keys, 0, pos);
-        this.keys = keys;
+        SelectionKey[] lKeys = new SelectionKey[this.keys.length << 1];
```

Comment:
It's the norm I followed everywhere in the code for last 3.5 years. Once you know `l` stands for `local`, you also know that it's named that way cause it's shadowing a variable without l prefix in the outer scope. It has actually helped improve code readability and maintainability. If also forces others to use more meaningful names than just shadowing other variables.